frontend/src/scenes/session-recordings/sessionRecordingLogic.ts

```diff
@@ -147,6 +155,28 @@ export const sessionRecordingLogic = kea<sessionRecordingLogicType>({
                 }
             },
         },
+        sessionEvents: {
+            loadEvents: async ({ url }) => {
+                if (!values.eventsApiParams) {
+                    return values.sessionEvents
+                }
+                // Use `url` if there is a `next` url to fetch
+                const startTime = performance.now()
+                const apiUrl = url || `api/projects/${values.currentTeamId}/events?${toParams(values.eventsApiParams)}`
+                const response = await api.get(apiUrl)
+
+                eventUsageLogic.actions.reportRecordingEventsFetched(
```

Comment:
I wonder if we want to send this event when all events are loaded - not when each batch of events loads?